README.md

```diff
@@ -49,6 +49,55 @@ const client = Binance({
 })
 ```
 
+### Browser vs Node.js
+
+This library works in both browsers and Node.js environments:
+
+**Public API calls**: Work in both environments
+```js
+const client = Binance()
+const time = await client.time()
+const prices = await client.prices()
+```
+
+**Authenticated API calls**: Work in both, but **NOT recommended in browsers** for security
+```js
```

Comment:
@pcriadoperez Not sure this is a good advise I think it depends a lot from case to case